src/Commands/Server/ToolOperations.cs

```diff
@@ -114,12 +114,17 @@ private async ValueTask<CallToolResponse> OnCallTools(RequestContext<CallToolReq
             var commandResponse = await command.ExecuteAsync(commandContext, commandOptions);
             var jsonResponse = JsonSerializer.Serialize(commandResponse, ModelsJsonContext.Default.CommandResponse);
 
-            return new CallToolResponse
+            return new CallToolResult
             {
                 Content = [
-                    new Content {
-                        Text = jsonResponse,
-                        MimeType = "application/json" }],
+                    new EmbeddedResourceBlock
+                    {
+                        Resource = new TextResourceContents
```

Comment:
I guessed here as to what behavior was desired. The spec does not support providing a MIME type with TextContent, so either this needs to be Text with no MIME type, or it needs to be an embedded resource with a TextResource. I went with the latter.